runbot_custom_demo_data/runbot_build.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Samuel Lefever
+#    Copyright 2015 Niboo SPRL
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import os
+import time
+import logging
+import openerp
+from openerp.fields import Boolean
+from openerp.models import api, Model
+
+_logger = logging.getLogger(__name__)
+
+
+def grep(filename, string):
```

Comment:
You can use original [grep](https://github.com/odoo/odoo-extra/blob/80722c03def52c8c1876d78ef3e805531562e8cb/runbot/runbot.py#L69) method with
`from openerp.addons.runbot.runbot import grep`